indexing-lambda/config/solr/core/conf/schema.xml

```diff
@@ -0,0 +1,36 @@
+<schema>
+    <fields>
+        <!--  Business logic fields -->
+        <field name="id" type="string" indexed="true" stored="true" required="true" multiValued="false"/>
+        <field name="author" type="text_general" indexed="true" stored="true"/>
+        <field name="title" type="text_general" indexed="true" stored="true"/>
+        <field name="description" type="text_general" stored="false"/>
+        <field name="tags" type="text_general" stored="false" multiValued="true"/>
+        <field name="publication_date" type="datetime" stored="false"/>
+
+        <!-- Service fields-->
+        <field name="_version_" type="plong" multiValued="false"/>
+    </fields>
+
+    <uniqueKey>id</uniqueKey>
+
+    <types>
+        <fieldType name="string" class="solr.StrField" sortMissingLast="true" docValues="true" />
+        <fieldType name="plong" class="solr.LongPointField" docValues="true"/>
+        <fieldType name="text_general" class="solr.TextField" positionIncrementGap="10"/>
+        <fieldType name="datetime" class="solr.TrieDateField" omitNorms="true">
+            <analyzer type="index">
+                <tokenizer class="solr.StandardTokenizerFactory"/>
+                <filter class="solr.StopFilterFactory" words="bad-words.txt" ignoreCase="true"/>
+                <filter class="solr.LowerCaseFilterFactory"/>
+                <filter class="solr.EnglishPossessiveFilterFactory"/>
+            </analyzer>
+            <analyzer type="query">
+                <tokenizer class="solr.StandardTokenizerFactory"/>
+                <filter class="solr.StopFilterFactory" words="bad-words.txt" ignoreCase="true"/>
+                <filter class="solr.LowerCaseFilterFactory"/>
+                <filter class="solr.EnglishPossessiveFilterFactory"/>
+            </analyzer>
```

Comment:
Since indexing and query-time analyzers are identical, you don't need to split them. Remove one of them and also remove type attribute from the retained one

Reply:
deleted